tests/mainnet.rs

```diff
@@ -14,10 +14,10 @@ pub mod common;
 
 #[test]
 fn mainnet_blocks_from_rpc() {
-    let rpc_url: Url = std::env::var("RPC_URL")
-        .unwrap_or("https://eth.llamarpc.com".to_string())
-        .parse()
-        .unwrap();
+    let rpc_url = match std::env::var("RPC_URL") {
+        Ok(value) if !value.is_empty() => value.parse().unwrap(),
```

Comment:
When and where it would exist but be empty? We need practical cases to justify more code & logic.

Reply:
Without this change, CI always fail when I make any PRs.

https://github.com/risechain/pevm/actions/runs/9108887873/job/25040598978?pr=78

Reply:
I see. We can push a step further to fallback when `RPC_URL` fails to parse:
```rust
    let rpc_url = std::env::var("RPC_URL")
        .ok()
        .and_then(|value| value.parse().ok())
        .unwrap_or(Url::parse("https://eth.llamarpc.com").unwrap());
```
But I guess it's okay to panic for the developer to double-check the value?